packages/mux-video/src/index.ts

```diff
@@ -501,6 +559,19 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
             .catch(() => console.error(`Unable to load or parse metadata JSON from metadata-url ${newValue}!`));
         }
         break;
+      case Attributes.STREAM_TYPE:
+        // If the newValue is unset
+        if (newValue == null || newValue !== oldValue) {
+          this.dispatchEvent(new CustomEvent('streamtypechange', { composed: true, bubbles: true }));
+        }
+        break;
+      case Attributes.TARGET_LIVE_WINDOW:
+        if (newValue == null || newValue !== oldValue) {
+          this.dispatchEvent(
+            new CustomEvent('targetlivewindowchange', { composed: true, bubbles: true, detail: this.targetLiveWindow })
```

Comment:
I wouldn't add `detail` here because that will never be added to the native HTML video tag